kong/plugins/basic-auth/utils.lua

```diff
@@ -0,0 +1,40 @@
+local resty_string = require "resty.string"
+local resty_sha1 = require "resty.sha1"
```

Comment:
Correct, there is no way to unit test inside functionality, which include - Openresty modules. 
imho, in addition to the current test suite, we need new test suite that could test inside modules. maybe we could use resty-cli for this.
for this particular case, i can include self-contained sha1 lib (or lua rock), so that i will not require resty module.

if this should be at the DAO level (how cassandra specific?), when we add support for other databases, any plugin will need to have its own sha1 (and upcoming encryption) implementations ?